ccdproc/combiner.py

```diff
@@ -447,6 +447,73 @@ def average_combine(self, scale_func=ma.average, scale_to=None,
         # return the combined image
         return combined_image
 
+    def sum_combine(self, sum_func=ma.sum, scale_to=None,
+                        uncertainty_func=ma.std):
+        """
+        Sum combine together a set of arrays.
+
+        A `~ccdproc.CCDData` object is returned with the data property
+        set to the sum of the arrays. If the data was masked or any
+        data have been rejected, those pixels will not be included in the
+        sum. A mask will be returned, and if a pixel has been
```

Comment:
Maybe include a sentence that if a re-scaled average sum is needed one should do `average_combine` and multiply it with the number of images combined.